### DISCUSSION TITLE
Import/Export issue labels to other repositories

### DISCUSSION THREAD

**Discussion by PatheticMustan** [community/community#10638] [Projects and Issues]
Many of my repositories have similar issue labels. As of now, there seems to be no way to export/import these.
Especially for personal repositories, or existing repos. I see default labels exists for organizations, would it be possible to also add this for personal accounts as well?

**Comment by ghost** [comment]
Hi, I too would like to see this new feature. We could imagine being able to export them in JSON format, then import them in another repository. A bit like this workaround: https://blog.avneesh.tech/how-to-add-labels-from-a-github-repo-to-another.

**Reply by O957** [reply]
The link seems to not be working.

Any updates on this?

**Comment by PatheticMustan** [comment]
It would be extremely helpful if there was a way to quickly import/export labels. If this is not the correct place, where else would I post a feature request?
